### ISSUE TITLE
Wrap Lagoon CLI commands with `lagoon()` function in deploy-lagoon.sh

### ISSUE THREAD

**Issue by AlexSkrypnyk** [drevops/scaffold#1242]
Similar to how we wrap `drush` commands with `drush()` function in `provision.sh`, we need to wrap `lagoon` with `lagoon()` to make it easy to read the code.

So,

```
lagoon --force --skip-update-check -i "${DREVOPS_DEPLOY_SSH_FILE}" -l "${DREVOPS_DEPLOY_LAGOON_INSTANCE}"delete environment -p "${LAGOON_PROJECT}" -e "${DREVOPS_DEPLOY_BRANCH}"
```

becomes

```
lagoon delete environment -e "${DREVOPS_DEPLOY_BRANCH}" || true
```

using 

```
lagoon() { lagoon --force --skip-update-check -i "${DREVOPS_DEPLOY_SSH_FILE}" -l "${DREVOPS_DEPLOY_LAGOON_INSTANCE}" -p "${LAGOON_PROJECT}" "$@";}
```

Need to do this in:

- `deploy-lagoon.sh`
- `task-custom-lagoon.sh`

**Comment by richardgaunt** [comment]
https://github.com/drevops/scaffold/pull/1244 fixes this issue I believe - happy to undertake more work if required

**Comment by AlexSkrypnyk** [comment]
@richardgaunt It does! There was no `closes #1242` comment in that PR so _this_ issue did not auto-close

Thank you so much for contributing!
